openslides/motions/models.py

```diff
@@ -777,6 +507,42 @@ def get_paragraph_based_amendments(self):
         return list(filter(lambda amend: amend.is_paragraph_based_amendment(), self.amendments.all()))
 
 
+class MotionCommentField(RESTModelMixin, models.Model):
```

Comment:
Please do not use "Field" in the name of the class because it can be mixed up with a Django Field (like TextField etc.). Proposal: MotionCommentSection